applications/nrf_desktop/src/hw_interface/battery_meas.c

```diff
@@ -25,7 +25,7 @@
 #include <zephyr/logging/log.h>
 LOG_MODULE_REGISTER(MODULE, CONFIG_DESKTOP_BATTERY_MEAS_LOG_LEVEL);
 
-#define ADC_DEVICE_NAME		DT_LABEL(DT_NODELABEL(adc))
+#define ADC_DEVICE		DT_NODELABEL(adc)
```

Comment:
For consistency with other similar definitions in tree, this symbol name should not end with `_DEVICE`, but rather with `_NODE`. But perhaps it would be better to use `DT_NODELABEL` directly in `DEVICE_DT_GET` below.

Reply:
renamed to `_NODE`. This module needs a 2nd pass cleanup with adc-dt-spec, we can then remove all `ADC_` defines.